### PR TITLE
5537 Add the population and housing facts to the map selection box

### PR DESCRIPTION
**Pull request by dhochbaum-dcp**
### Summary
Adds the summary totals to the map selection box.
image

#### Tasks/Bug Numbers
 - Along with a corresponding [pr in labs-factfinder-api](https://github.com/NYCPlanning/labs-factfinder-api/pull/199), completes [AB#5537](https://dev.azure.com/NYCPlanning/cc280b0d-40a0-4689-b852-2e6247f1af50/_workitems/edit/5537)

### CODE REVIEW

File: app/utils/fetch-selection-summary.js
Comment (TylerMatteo) [inline app/utils/fetch-selection-summary.js:8]: @godfreyyeung do you think this should me in a new Model? My understanding was that any async calls in Ember should be encapsulated in a Model but I'm not necessarily opposed to forgoing that for this functionality.

File: app/services/selection.js
Comment (TylerMatteo) [inline app/services/selection.js:285]: Just a note - I don't have a problem with this because this service already has this imperative `this.set` pattern throughout it but sure do with this was handled declaratively with some sort of "tracked" property.

**Comment by TylerMatteo** [comment]
Looks good to me other than _maybe_ moving the data fetching into a Model. I'm going to hold off on approving until the API PR is merged, since changes there might necessitate some changes here.

**Comment by dhochbaum-dcp** [comment]
> 

API PR has been merged, and does not require updates to this PR.

File: app/utils/fetch-selection-summary.js
Comment (godfreyyeung) [reply app/utils/fetch-selection-summary.js:8]: in this case it's a lot of overhead and work to rework it into a model. It would require the API to deliver JSON-API and we'd have to rework the selection service, and all dependents of the selection service

**Review by TylerMatteo — APPROVED** [review]

**Pull request merged**
